web/js/projectManagement.js

```diff
@@ -1100,18 +1100,21 @@ Ext.onReady(function(){
         listeners: {
             'write': function (store, action, result, res, rs) {
                 App.setAlert(true, "Projects Changes Saved");
-
+                
                 // select newly added row and scroll to it
                 var rowIndex = store.indexOf(rs);
                 projectGrid.getSelectionModel().selectRow(rowIndex);
                 projectGrid.getView().focusRow(rowIndex);
             },
-            'exception': function(){
-                App.setAlert(false, "Some Error Occurred While Saving The Changes");
-            },
+            'exception': function(proxy, type, action, exception, res){
+                var parser = new DOMParser();
+                var errorDoc = parser.parseFromString(res.responseText, "text/xml");
+                var errorMessage = errorDoc.getElementsByTagName("error")[0].childNodes[0].nodeValue;
```

Comment:
I'm not sure how the format of the XML response is but is it possible to have multiple error elements? In this case we may need to be more specific to be sure we are displaying all the errors, but I'm happy to fetch this and test it next week as well as I'm not completely sure how it works.

Reply:
We should be able to. Since we are sending back xml to the front end, I think we could add as many <error> items to the object as needed and then just parse them out to the alert. I'll test the js here with multiple <error> elements; I probably need to modify the last bit of code since it's taking the first items out of the arrays instead of looping.